pom.xml

```diff
@@ -124,6 +124,61 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <version>0.9.0-SNAPSHOT</version>
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <title>Kafka Connect Elasticsearch</title>
+                            <logo>logos/elasticsearch.png</logo>
```

Comment:
As with the other PRs, where is the documentation that says we can use this logo?